src/ExperimentTable.js

```diff
@@ -145,16 +158,27 @@ class ExperimentTable extends React.Component {
 
     const displayedFields = tableHeader.map(header => header.dataParam)
 
-    //first we filter according to table headers
-    //then on those filtered experiments we further filter them in next step according to search box text
-    //and finally we filter the experiments filtered in previous two steps according to selected dropdowns
-    const filteredExperiments = this.filter(this.sort(aaData), tableHeader).filter(data =>
-      data && Object.keys(data).map(key =>
-        displayedFields.includes(key) ? data[key] : null).some(value =>
-        value && value.toString().toLowerCase().includes(selectedSearch))).filter(experiment =>
-      selectedDropdownFilters.every(filter =>
-        filter ? this.containsValue(experiment, filter.value) : true
-      ))
+    const filteredExperiments =
+      //first we filter experiments according to table headers
+      this.filter(this.sort(aaData), tableHeader)
+      //...then we further filter them according to search box text
+        .filter(experiment =>
+          experiment &&
+          Object
+            .keys(experiment)
+            .map(key =>
+              displayedFields.includes(key) ? experiment[key] : null)
+            .some(value =>
+              value &&
+              value
+                .toString()
+                .toLowerCase()
+                .includes(selectedSearch)))
```

Comment:
I just noticed that if you do `value.toLowerCase()` then you should do `selectedSearch.toLowerCase()` too.